packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -309,6 +309,7 @@ const Autocomplete = <
       renderInput={renderInput}
       isOptionEqualToValue={getIsOptionEqualToValue}
       translate={translate}
+      open={true}
```

Comment:
This opens the Autocomplete on load, right? Will that be the default behavior for the component with this?

Reply:
Oh shoot, I need to remove that. I had it in there to make styling/debugging easier. Good catch :)

Reply:
Removed [here](https://github.com/okta/odyssey/pull/2437/commits/125143491598f1618e24fe484bbdf1bf75a1d4b4)